Ship_Game/Universe/UniverseScreen/UniverseScreen.cs

```diff
@@ -206,20 +206,22 @@ void ResetLighting(bool forceReset)
             AddLight("Global Back Light", new Vector2(0, 0), 0.6f, globalLightRad, Color.White, +globalLightZPos, fillLight: false, shadowQuality: 0f);
 
             foreach (SolarSystem system in UState.Systems)

```

Comment:
there seems nothing inside this foreach

Reply:
It was not changed so you see  trailing code after the actual change:
this is the code:
```
           foreach (SolarSystem system in UState.Systems)
                ResetSolarSystemLights(system);
```